README.md

```diff
@@ -183,6 +183,11 @@ In order to install Valkey binaries into /usr/local/bin, just use:
 You can use `make PREFIX=/some/other/directory install` if you wish to use a
 different destination.
 
+_Note_: For compatibility with Redis, we create symlinks from the Redis names (redis-server, redis-cli, etc.) to the Valkey binaries created by the `make install`.
+The symlinks are created in same directory as the the Valkey binaries.
+The symlinks are removed when using `make uninstall`.
+The creation of the symlinks can be skipped by setting the environment variable `CONFIGURE_REDIS_SYMLINKS=false`.
```

Comment:
```suggestion
The creation of the symlinks can be skipped by setting the environment variable `INSTALL_REDIS_SYMLINKS=no`.
```

The other makefile booleans use yes/no. Let's follow that.

Reply:
Good point. It's not configuring wither though.. USE_REDIS_SYMLINKS?

Another possibility would be to remove the symlinks in uninstall regardless of this variable, but instead we can check that they are symlinks before deleting them?

Reply:
> USE_REDIS_SYMLINKS

I think this is simpler.